cds_dojson/marc21/fields/videos/video.py

```diff
@@ -140,3 +141,65 @@ def copyright(self, key, value):
         'year': value.get('g'),
         'message': value.get('f'),
     }
+
+
+@model.over('_files', '^8567_')
+@for_each_value
+@filter_values
+def _files(self, key, value):
+    """File list."""
+    def get_key(value):
+        if value.get('d'):
+            return value.get('d').split('\\')[-1]
+        else:
+            return os.path.basename(value.get('u'))
+
+    def get_context_type(value):
+        if value.get('d'):
+            return 'master', 'video'
+        if value.get('y').startswith('thumbnail'):
+            return 'frame', 'image'
+        if 'kbps maxH' in value.get('y'):
```

Comment:
I didn't check enough videos to be sure that one can find always 'kbps maxH' for every record. Did you? :) If it is always like that, then I am good with it 👍